### PR TITLE
Added 'editable:vertex:new' event

### LINKED ISSUE #102: enhancement: add editable:vertex:new event
I want to execute a function each time a new vertex is added to a polyline:
1. when clicking on map while continuing backward or forward
2. when clicking a middlemarker to convert it to a vertex

Could there be a `editable:vertex:new` event in such cases?
For 1. I can capture the `editable:drawing:click` event, but at that time the vertex is not yet added to the Polyline.
For 2. I can capture the `editable:vertex:dragend` event, but I think it is weird to get same event for a new vertex and a moved one. Or use `editable:middlemarker:mousedown`, but at that time vertex is not added to polyline yet.
Thanks

### PR DESCRIPTION
**Pull request by opoto**
Proposal to fix #102

### CODE REVIEW

File: test/PolylineEditor.js [hunk 1]
```diff
@@ -551,6 +551,18 @@ describe('L.PolylineEditor', function() {
             layer.remove();
         });
 
+        it('should fire editable:vertex:new ', function () {
```
Comment (yohanboniface) [inline test/PolylineEditor.js]: Would you mind adding one similar test for Polygon? There are subtle differences between PolylineEditor and PolygonEditor, so it's better always checking for both.
Also, one quick test for checking that the event is fired when dragging a middle marker?

**Comment by opoto** [comment]
Added suggested tests

**Comment by yohanboniface** [comment]
> Added suggested tests

Excellent, thanks!
One last thing: I'm thinking it may be worth adding one more test to check that the event is *not* fired when vertexes are created when enabling editing.
I'm sure this is not the case here, but this kind of tests will help in a future refactoring.
And then we are good to merge and release! :)
Thanks :)

**Comment by opoto** [comment]
done

**Comment by opoto** [comment]
Any update on this one ? Thanks

**Comment by yohanboniface** [comment]
Sorry for the lag! :(

**Pull request merged**